corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -305,6 +305,22 @@
                         </select>
                     </div>
                 </div>
+
+                {% if app.grid_display_for_some_modules %}
+                <div class="form-group">
+                    <label class="col-sm-2 control-label">
+                        {% trans "Display Style" %}
```

Comment:
Is there any description of grid view on confluence? (Jeremy or Amelia would know.) Would be nice to have a help bubble that links to it - this is one of those features that's obviously great if you can see a sample image, but hard to describe in words what it does.